index.html

```diff
@@ -6,12 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Most Followed GitHub Users</title>
     <link rel="preconnect" href="https://cdn.jsdelivr.net" crossorigin />
-    <link
-      rel="stylesheet"
-      type="text/css"
-      href="https://rawcdn.githack.com/kerolloz/Fomantic-UI/0b4aa1341a7e98bc4b1193601d5a62e4b59dc8d9/dist/semantic.min.css?min=1"
-    />
-
+    
+	<!-- You MUST include jQuery 3+ before Fomantic -->
+	<script src="https://fastly.jsdelivr.net/npm/jquery@3.6.3/dist/jquery.min.js"></script>
+	<link rel="stylesheet" type="text/css" href="https://fastly.jsdelivr.net/npm/fomantic-ui@2.9.2/dist/semantic.min.css">
+	<script src="https://fastly.jsdelivr.net/npm/fomantic-ui@2.9.2/dist/semantic.min.js"></script>
+    
```

Comment:
Thank you @aboelkassem for the PR ✨ 

I guess we only need to include the CSS files (get rid of the jquery and .js scripts)

```suggestion
	<link rel="stylesheet" type="text/css" href="https://fastly.jsdelivr.net/npm/fomantic-ui@2.9.2/dist/semantic.min.css">    
```

Reply:
removed